tensorflow/inference/docker/build_artifacts/sagemaker/serve.py

```diff
@@ -218,35 +219,32 @@ def _setup_gunicorn(self):
                         raise ChildProcessError("failed to install required packages.")
 
         gunicorn_command = (
-            "gunicorn -b unix:/tmp/gunicorn.sock -k {} --chdir /sagemaker "
+            "python3 /sagemaker/python_service.py -b unix:/tmp/gunicorn.sock -k {} --chdir /sagemaker "
```

Comment:
One question I have about gunicorn CLI vs doing it with `.run()` python API: is there any different in the ability of gunicorn to restart workers automatically when they die? Just check on that before merge...

Reply:
Hmm, I didn't see any mention or warning about difference of using cli vs python API in their doc : https://docs.gunicorn.org/en/stable/custom.html#direct-usage-of-existing-wsgi-apps